<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateScaledShapeModelDialog</class>
 <widget class="QDialog" name="CreateScaledShapeModelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(247, 255, 171);
	color: rgb(0, 0, 0);
border-style:solid;
border-width:2px;
border-radius:5px;
}
QLabel{
background-color: rgb(170, 255, 127);
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
min-width: 155px;
}
QComboBox{
background-color: rgb(180, 180, 180);
min-height: 30px;
}
QDoubleSpinBox{
background-color: rgb(255, 255, 255);
min-height: 30px;
}
QPushButton{
background-color: rgb(255, 170, 0);
border-color: rgb(0, 0, 0);
border-style:solid;
border-width:2px;
border-radius:5px;
font:15px &quot;微软雅黑&quot;;min-width: 70px;
}
QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(255, 200, 157, 255), stop:0.857955 rgba(157, 213, 221, 255));
}
QPushButton:pressed{
background-color: rgb(47, 141, 69);
border-color: rgb(179, 179, 179);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>金字塔层数</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>模板旋转的起始角度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="minimum">
         <double>-7.000000000000000</double>
        </property>
        <property name="maximum">
         <double>7.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>-1.570000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>模板旋转的角度范围</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
        <property name="minimum">
         <double>-7.000000000000000</double>
        </property>
        <property name="maximum">
         <double>7.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>3.140000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>旋转角度的步长</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>行列最小缩放</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>行列最大缩放</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>行列的步长</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>模板创建方法</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.0175</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.0349</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.0524</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.0698</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.0873</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.900000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.500000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.05</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.1</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboBox_5">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>none</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>point_reduction_low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>point_reduction_medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>point_reduction_high</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pregeneration</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>no_pregeneration</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>匹配方法设置</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboBox_7">
        <item>
         <property name="text">
          <string>use_polarity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ignore_global_polarity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ignore_local_polarity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ignore_color_polarity</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>设置对比度</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="comboBox_6">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>auto_contrast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>auto_contrast_hyst</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>auto_min_size</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>120</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>140</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>160</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>设置最小对比度</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="comboBox_8">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(85, 170, 255);</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>439</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
